R/summarystatsbayesianztest.R

```diff
@@ -0,0 +1,404 @@
+#
+# Copyright (C) 2013-2018 University of Amsterdam
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+SummaryStatsBayesianZTest <- function(jaspResults, dataset = NULL, options, ...) {
+
+  # parse formulas
+  options <- .bayesianZTestParseOptions(options)
+
+  # get data
+  .bayesianZTestGetData(jaspResults, options)
+
+  # get results
+  .bayesianZTestSummaryTable(jaspResults, options)
+
+  if (options[["plotPriorAndPosterior"]])
+    .bayesianZTestPriorPosteriorPlot(jaspResults, options)
+
+  if (options[["plotBayesFactorRobustness"]])
+    .bayesianZTestRobustnessPlot(jaspResults, options)
+
+  return()
+}
+
+
+.bayesianZTestsDependencies   <- c(
+  "dataType", "dataEs", "dataSe", "dataLCi", "dataUCi",
+  "hypothesis", "priorMean", "priorSd"
+  )
+.bayesianZTestParseOptions       <- function(options) {
+
+  options[["dataEs"]]  <- jaspBase:::.parseRCodeInOptions(options[["dataEs"]])
+  options[["dataSe"]]  <- jaspBase:::.parseRCodeInOptions(options[["dataSe"]])
+  options[["dataLCi"]] <- jaspBase:::.parseRCodeInOptions(options[["dataLCi"]])
+  options[["dataUCi"]] <- jaspBase:::.parseRCodeInOptions(options[["dataUCi"]])
+
+  options[["priorMean"]] <- jaspBase:::.parseRCodeInOptions(options[["priorMean"]])
+  options[["priorSd"]]   <- jaspBase:::.parseRCodeInOptions(options[["priorSd"]])
```

Comment:
```suggestion
  options[["dataEs"]]  <- jaspBase::.parseRCodeInOptions(options[["dataEs"]])
  options[["dataSe"]]  <- jaspBase::.parseRCodeInOptions(options[["dataSe"]])
  options[["dataLCi"]] <- jaspBase::.parseRCodeInOptions(options[["dataLCi"]])
  options[["dataUCi"]] <- jaspBase::.parseRCodeInOptions(options[["dataUCi"]])

  options[["priorMean"]] <- jaspBase::.parseRCodeInOptions(options[["priorMean"]])
  options[["priorSd"]]   <- jaspBase::.parseRCodeInOptions(options[["priorSd"]])
```
`.parseRCodeInOptions` is [exported](https://github.com/jasp-stats/jaspBase/blob/dec4a6fef22a9ee75688a65bdff0c7cb9f7f209c/NAMESPACE#L28) so you only need `::`.